### PR TITLE
DOC: State space: release notes and documentation.

### PR DESCRIPTION
**Pull request by ChadFulton**
Updates the release notes and documentation to make it consistent with the recent changes.

### CODE REVIEW

**Comment by josef-pkt** [comment]
@ChadFulton  This contains #2515 in the first two commits.

Can I merge this together in this PR (making #2515) obsolete, or do you want to do an interactive rebase to drop the first two commits from this PR?

From I quick look at didn't see anything else. I doubt I'll have time today to try out the notebooks, and will merge as it is.

**Comment by ChadFulton** [comment]
Thanks for catching that, sorry. I'll do the rebase to remove those so that the history is cleaner.

File: docs/source/release/version0.7.rst [hunk 1]
```diff
@@ -72,29 +73,44 @@ local level model can be specified and estimated:
...
    # Fit the model with maximum likelihood estimation
+   mod = LocalLevel(data['volume'])
    res = mod.fit()
    print res.summary()
 
 The documentation and example notebooks provide further examples of how to
-create ad-hoc models (as above) as well as more complex models. Included in
-this release is a full-fledged model making use of the state space
-infrastructure to estimate SARIMAX models. See below for more details.
+formulating state space models. Included in this release is a full-fledged
```
Comment (josef-pkt) [inline docs/source/release/version0.7.rst]: "formulating" sounds incorrect "formulate" ?

**Comment by josef-pkt** [comment]
merging, diff of notebooks is useless, the version upgrade noise will be only once.

File: docs/source/release/version0.7.rst (hunk 1 above)
Comment (ChadFulton) [inline docs/source/release/version0.7.rst]: Thanks, you're right, I've submitted a new PR to fix this #2539

**Pull request merged**